src/components/ListItem.jsx

```diff
@@ -27,10 +31,38 @@ export function ListItem({ item, listPath }) {
 		mutationFn: markAsPurchased,
 	});
 
+	const previousEstimate = getDaysBetweenDates(
+		dateNextPurchased,
+		dateLastPurchased,
+	);
+
+	function estimateAddedToNow() {
```

Comment:
I would export this function from `utils/dates.js` and turn this into a pure function

Reply:
Also from a readability standpoint I am having a hard time understanding what this function does.